api/Sources/Api/PairQL/Dashboard/Pairs/UserActivityFeed.swift

```diff
@@ -65,19 +65,20 @@ extension UserActivityFeed: Resolver {
       throw Abort(.badRequest)
     }
 
-    let user = try await context.verifiedUser(from: input.userId)
-    let userDeviceIds = try await user.devices(in: context.db).map(\.id)
+    let child = try await context.verifiedUser(from: input.userId)
+    let computerUserIds = try await child.computerUsers(in: context.db).map(\.id)
+    let iosDeviceIds = try await child.iosDevices(in: context.db).map(\.id)
 
     async let keystrokes = KeystrokeLine.query()
-      .where(.computerUserId |=| userDeviceIds)
+      .where(.computerUserId |=| computerUserIds)
       .where(.createdAt <= .date(before))
       .where(.createdAt > .date(after))
       .orderBy(.createdAt, .desc)
       .withSoftDeleted()
       .all(in: context.db)
 
     async let screenshots = Screenshot.query()
-      .where(.computerUserId |=| userDeviceIds)
+      .where(.or(.computerUserId |=| computerUserIds, .iosDeviceId |=| iosDeviceIds))
```

Comment:
here's the other spot (based on whether you're looking at all children in the dashboard, or just one)